projects/golang/go/1.18/rpmbuild/SPECS/golang.spec

```diff
@@ -108,7 +108,7 @@
 # Comment out go_prerelease and go_patch as needed
 %global go_api 1.18
 #global go_prerelease rc1
-%global go_patch 7
+%global go_patch 8
```

Comment:
What are these?

Reply:
NM. Got it.

Reply:
patch version hard-coded in the spec file. we should change them to use a variable for it that we derive from the git-tag but for the moment I'm just bumping it